src/pat/ajax/ajax.js

```diff
@@ -90,7 +90,13 @@ define([
                     error: onError,
                     success: onSuccess
                 };
-
+            // add nocache url parameter to prevent IE11 from caching.         
+            if ( cfg.url.indexOf("?") == -1 ) {
+                args.url = cfg.url + '?nocache=' + Date.now();
+            }
+            else {
+                args.url = cfg.url + '&nocache=' + Date.now();
+            }
```

Comment:
I'm a little bit worried about badly formatted URLs, which however do sometimes happen. For example a URL which ends in `?`. Could you please handle that one case?